lib/morse_translator.rb

```diff
@@ -38,6 +39,27 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+
+  end
+
+  def translate(message)
+    holder = []
+    morse = []
+    holder << message.chars
+    @morse_array = holder.flatten
```

Comment:
This doesn't need to be an instance variable

Reply:
A better way to get the individual characters would be `morse_array = holder.chars`